src/data-source.js

```diff
@@ -28,13 +28,15 @@ function dataSource(params) {
   }
 
   // expect the _ow_path to start with /https:// or /https%3a%2f%2f
-  if (path.startsWith('/https%3A%2F%2F')) {
+  if (path.startsWith('/https%3A%2F%')) {
     return new URL(decodeURIComponent(path.substring(1)));
   }
-  if (!path.startsWith('/https://')) {
+  if (!path.startsWith('/https:/')) {
     return null;
   }
-  const url = new URL(path.substring(1));
+  const url = new URL(path.substring(1)
+    // workaround: Adobe I/O Runtime messes up consecutive spaces in URLs
+    .replace(/^https:\/\/?([^/])/, 'https://$1'));
```

Comment:
can you add a test?

Reply:
added in https://github.com/adobe/helix-embed/pull/308/commits/9c94152fe0c63d7627176073a9d7f38e199ec979